source/payment_flow/index.html.md.erb

```diff
@@ -95,7 +95,7 @@ Your PSP will then take (‘capture’) the payment from your user’s bank acco
 
 If your user decides not to complete the payment or the payment fails, your user will see a GOV.UK Pay error page, which includes a **Continue** button. When your user selects this, they’ll be redirected to your `return_url`.
 
-If your user does not complete their payment within 90 minutes, it will expire. GOV.UK Pay will send a cancellation to the payment provider and the status of the payment will be [P0020](/api_reference/#api-error-codes).
+If your user does not complete their payment within 90 minutes, it will expire. GOV.UK Pay will send a cancellation to the payment provider and the status of the payment will be [P0020](/api_reference#responses).
```

Comment:
Should this be below?

```suggestion
If your user does not complete their payment within 90 minutes, it will expire. GOV.UK Pay will send a cancellation to the payment provider and the status of the payment will be [P0020](/api_reference#gov-uk-pay-api-error-codes).
```

Reply:
Yes, good spot!